test/unit/presenters/document_export_presenter_test.rb

```diff
@@ -149,6 +149,31 @@ class DocumentExportPresenterTest < ActiveSupport::TestCase
                  result.dig(:editions, 0, :fact_check_requests, 0)
   end
 
+  test "includes history information" do
+    user_1 = create(:user)
+    Edition::AuditTrail.whodunnit = user_1
+    edition = create(:edition)
+
+    user_2 = create(:user)
+    Edition::AuditTrail.whodunnit = user_2
+    edition.update!(change_note: "changed")
+
+    result = DocumentExportPresenter.new(edition.document).as_json
+    expected = [
+      { event: "create",
+        whodunnit: user_1.id.to_s,
```

Comment:
Did we agree to convert this to `int`?

Reply:
Yes - though we need to work out when the user changes are done.